File: sentimental-cash/cash.py
from cs50 import get_float

change = -1
coins = 0
while(change < 0):
    change = get_float("Change owed: ")

if(change >= 0.25):
    n = change / 0.25
    n = round(n)
    for i in range(n):
        coins+= 1
        change -= 0.25

if(change >= 0.10):
    n = change / 0.10
    n = round(n)
    for i in range(n):
        coins += 1
        change -= 0.10


print(coins)

File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    //Prompt the user for a positive float numbers
    float change;
    int coins;
    do
    {
        change = get_float("Change owed: ");
    }
    while (change < 0);

    //round the cents to the nearest penny
    int cents = round(change * 100);

    //Use the biggest coins available, while keep tracking the numbers of coins used
    for (coins = 0; cents >= 25; coins++)
    {
        cents = cents - 25;
    }

    for (coins = coins + 0; cents >= 10; coins++)
    {
        cents = cents - 10;
    }

    for (coins = coins + 0; cents >= 5; coins ++)
    {
        cents = cents - 5;
    }

    for (coins = coins + 0; cents >= 1; coins++)
    {
        cents = cents - 1;
    }

    //print the numbers of coins used
    printf("%i\n", coins);

}